Cargo.toml

```diff
@@ -54,12 +54,14 @@ bytes = "1.5"
 bytesize = "1.3"
 chrono = { version = "0.4.35", default-features = false }
 clap = "4"
+moka = "0.12.10"
```

Comment:
have you consider also exposing moka expiration policies TTL and TTI as env vars ? I was hoping that we can control both time and mem dimensions ... Moka looks great!

Reply:
Yeah was considering it.   But at least as this PR is, we should be able to avoid blowing up the memory usage which was the main goal of this PR.